console/console-init/ui/src/pages/AddressSpaceDetail/AddressSpaceDetailPage.tsx

```diff
@@ -168,6 +209,53 @@ export default function AddressSpaceDetailPage() {
     },
     onDelete: data => {
       setIsDeleteModalOpen(!isDeleteModalOpen);
+    },
+    onEdit: () => {
+      setIsEditModalOpen(true);
+      setAddressSpaceBeingEdited(addressSpace);
+    }
+  };
+
+  const handleCancelEdit = () => {
+    setIsEditModalOpen(false);
+    setAddressSpaceBeingEdited(null);
+  };
+  const handleSaving = async () => {
+    addressSpaceBeingEdited &&
+      (await client.mutate({
```

Comment:
we are using custom hook for mutation query i.e. useMutationQuery  for server error handling.
Replace this code with useMutaionQuery hook as we implemented in other page/form.